src/index.ts

```diff
@@ -14,11 +21,116 @@ export interface Env {
 	//
 	// Example binding to a Queue. Learn more at https://developers.cloudflare.com/queues/javascript-apis/
 	// MY_QUEUE: Queue;
+
+	DISCORD_TOKEN: string,
+	DISCORD_PUBLIC_KEY: string,
+	DISCORD_CLIENT_ID: string,
+	GITHUB_TOKEN?: string,
+	GUILD_ID?: string,
+	SUPPORT_CHANNEL?: string,
+	SUPPORT_AI_CHANNEL?: string,
+	SUPPORT_SQUAD_CHANNEL?: string,
+	STATS_SCHEDULE?: string,
+	SUPPORT_REDIRECT_SCHEDULE?: string,
+	ALGOLIA_INDEX?: string,
+	ALGOLIA_APP_ID?: string,
+	ALGOLIA_API_KEY?: string
+}
+
+export class JsonResponse extends Response {
+  constructor(body?: any, init?: ResponseInit) {
+    const jsonBody = JSON.stringify(body);
+    init = init || {
+      headers: {
+        'content-type': 'application/json;charset=UTF-8',
+      },
+    };
+    super(jsonBody, init);
+  }
 }
```

Comment:
I don't really get this `JsonResponse`. What is the reason we need this?

Reply:
I kinf of just copied that from the example. Will remove